Project-I-Social-Card/social-card/src/Header.js

```diff
@@ -8,7 +8,7 @@ function Header() {
 }
 
 function ImageThumbnail(){
-  return <div className="img">
+  return <div className="imgDiv">
     <img className="img" src="https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png" alt="lambda logo"/>
   </div>
```

Comment:
- Inaki great effort on the assignment. Not only You completed the Social Card but started working on the Calc. 
- Your code is clean and You are properly splitting up the code into multiple components. 
- Try and work more on the calculator - you could use .map() to generate new components and pass props to them. 
- Your git workflow is fluid and You did great job on this assingment so far 👍